main/main.cpp

```diff
@@ -1720,6 +1725,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 				OS::get_singleton()->print("Missing write-movie argument, aborting.\n");
 				goto error;
 			}
+#else
+			OS::get_singleton()->print("Missing `MovieWriter` module, aborting.\n");
```

Comment:
I'd put this at the top with `#ifdef` instead, `#ifdef`/`#else` is more readable